recipes-crosswalk/crosswalk/crosswalk_20.50.533.12.bb

```diff
@@ -455,8 +484,14 @@ do_install() {
     install -d ${D}${libdir}/xwalk/
     install -m 0755 ${S}/out/Release/xwalk ${D}${libdir}/xwalk/xwalk
     install -m 0644 ${S}/out/Release/icudtl.dat ${D}${libdir}/xwalk/icudtl.dat
-    install -m 0644 ${S}/out/Release/libffmpegsumo.so ${D}${libdir}/xwalk/libffmpegsumo.so
-    install -m 0644 ${S}/out/Release/xwalk.pak ${D}${libdir}/xwalk/xwalk.pak
+    install -d ${D}${libdir}/xwalk/lib/
+    install -m 0644 ${S}/out/Release/lib/libffmpeg.so ${D}${libdir}/xwalk/lib/libffmpeg.so
+    install -m 0644 ${S}/out/Release/*.pak ${D}${libdir}/xwalk/
+    install -m 0644 ${S}/out/Release/natives_blob.bin ${D}${libdir}/xwalk/
```

Comment:
According to https://github.com/crosswalk-project/crosswalk/blob/master/tools/installer/common/installer.include, there should be an *if* statement here.